### PR TITLE
AB2-711 Add top sites onboarding

### PR DESCRIPTION
**Pull request by jayeshsolanki93**

### CODE REVIEW

File: l10n/de/mobile/android/base/cliqz_strings.dtd [hunk 1]
```diff
@@ -8,6 +8,9 @@
 <!-- Onboarding -->
 <!ENTITY firstrun_urlbar_subtext3 "&firstrun_urlbar_subtext;">
 
+<!-- Top Sites -->
+<!ENTITY activity_stream_empty_toppanel "Hier erscheinen deine meistbesuchten Seite">
```
Comment (spacifici) [inline l10n/de/mobile/android/base/cliqz_strings.dtd]: We do not add strings in the translations

Comment (jayeshsolanki93) [reply l10n/de/mobile/android/base/cliqz_strings.dtd]: Removed this and fixed the conflict :+1:

File: mozilla-release/mobile/android/base/java/org/mozilla/gecko/activitystream/homepanel/stream/TopPanelRow.java [hunk 2]
```diff
@@ -27,7 +27,9 @@
 
     /* Cliqz Start */
     private final PreferenceManager mPreferenceManager;
-    /* Cliqz End*/
+
+    private final View emptyTopSitesOnboarding;
```
Comment (ravjit-cliqz) [inline mozilla-release/mobile/android/base/java/org/mozilla/gecko/activitystream/homepanel/stream/TopPanelRow.java]: mEmpty

Comment (spacifici) [reply mozilla-release/mobile/android/base/java/org/mozilla/gecko/activitystream/homepanel/stream/TopPanelRow.java]: nope, it is `final`

**Pull request merged**